rtpsniff.c

```diff
@@ -87,12 +89,18 @@ int main(int argc, char const *const *argv) {
     /* Finish/close open stuff */
     sniff_release(&memory.rtphash[0]);
     sniff_release(&memory.rtphash[1]);
-	
+
     out_close();
     pcap_close(handle);
     return 0;
 }
 
+void exit_error(char* errbuf) {
+	fprintf(stderr, "RTPSniff: Initialization failed or bad command line "
+		        "options. See -h for help:\n%s\n", errbuf);
+	exit (EXIT_FAILURE);
```

Comment:
Gaarne n*4 spaties in plaats van tabs. De rest van de file doet ook 4 spaties namelijk.